lte/gateway/docker/docker-compose.yaml

```diff
@@ -141,7 +141,10 @@ services:
       - seccomp:unconfined
     environment:
       MAGMA_PRINT_GRPC_PAYLOAD: 0
-    command: /usr/local/bin/sctpd
+    command: >
+      sh -c "/usr/local/bin/magma-bridge-reset.sh &&
+            /etc/init.d/openvswitch-switch load-kmod &&
+            /usr/local/bin/sctpd"
```

Comment:
magma-bridge-reset.sh takes care of loading kmod, there is no need to do it here.

Reply:
@pshelar actually, magma-bridge-reset.sh call force-reload-kmod. This is would not work from within the container.

Reply:
why `openvswitch-switch load-kmod` works, but `force-reload-kmod` does not?